README-template.md

```diff
@@ -1,115 +0,0 @@
-# Frontend Mentor - Interactive card details form solution
```

Comment:
**question:** Why do you add the readme twice?

Reply:
It's a standard readmes when I download it from the site but it's true I should clean that up.